src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+interface ITextFieldProps {
+  customClassNames?: string;
+  value?: string;
+  label?: string;
+  type: string;
```

Comment:
It might be helpful to maybe use a union type for this field just to restrict it to the values you expect to receive, vs free form string?
```suggestion
  type: "text" | "number" | "url";
```

Reply:
Better yet it might be a good idea to create another TextInput component and restrict it to only handling numbers? `NumberTextFieldComponent` ?